docs/environment.yml

```diff
@@ -1,10 +1,11 @@
 name: qtconsole_docs
 channels:
+  - astrofrog
```

Comment:
Thanks @takluyver. Just a clarification on the front end config options: were there items missing or detail on the items (i.e. options) missing or both?